etc/scdaq/scdaq.conf

```diff
@@ -59,9 +61,10 @@ doZS:no
 ##
 ################################################################################
 
-output_filename_prefix:scout_pass_through
+output_filename_prefix:scout_GMT
 
 output_filename_base:/fff/BU0/ramdisk/scdaq
```

Comment:
I realise you haven't touched this line, but I wonder if we should comment this out too. Probably depends on how soon we get the connection to the surface on -34..

Reply:
in any case for now it's different for the two machines so same approach as the two previous ones applies